From f56bebd85a5a3eb18da74fb3f8cf6a8026cbb326 Mon Sep 17 00:00:00 2001
From: "Kothapeta, BikshapathiX" <bikshapathix.kothapeta@intel.com>
Date: Thu, 8 Apr 2021 15:34:38 +0530
Subject: [PATCH] drm: remove drm 1.0 packages

Even when DRM version is set to 1.3, drm 1.0 service is running
which leads to CTS, VTS failure.

Fix the issue by removing drm 1.0 packages.

Tracked-On: OAM-96821
Signed-off-by: Kothapeta, BikshapathiX <bikshapathix.kothapeta@intel.com>
---
 groups/default-drm/true/product.mk | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/groups/default-drm/true/product.mk b/groups/default-drm/true/product.mk
index 67d7d20..a39255a 100644
--- a/groups/default-drm/true/product.mk
+++ b/groups/default-drm/true/product.mk
@@ -1,5 +1,3 @@
 #only enable default drm service
-PRODUCT_PACKAGES += android.hardware.drm@1.0-service \
-                    android.hardware.drm@1.0-impl \
-                    android.hardware.drm@1.3-service.clearkey
+PRODUCT_PACKAGES += android.hardware.drm@1.3-service.clearkey
 
-- 
2.17.1

